Makefile

```diff
@@ -23,25 +23,33 @@ clean:
 
 makemessages: /tmp/template.pot
 
-translations: ./example/translations.json
+translations: ./$(OUTPUT_DIR)/translations.json
 
 # Create a main .pot template, then generate .po files for each available language.
+# Thanx to Systematic: https://github.com/Polyconseil/systematic/blob/866d5a/mk/main.mk#L167-L183
 /tmp/template.pot: $(GETTEXT_HTML_SOURCES)
+# `dir` is a Makefile built-in expansion function which extracts the directory-part of `$@`.
+# `$@` is a Makefile automatic variable: the file name of the target of the rule.
+# => `mkdir -p /tmp/`
 	mkdir -p $(dir $@)
 	which gettext-extract
+# Extract gettext strings from templates files and create a POT dictionary template.
 	gettext-extract --quiet --output $@ $(GETTEXT_HTML_SOURCES)
-	xgettext --language=JavaScript --keyword=i18n --keyword=npgettext:1c,2,3 \
-		 --from-code=utf-8 --sort-output --join-existing --no-wrap \
+# Extract gettext strings from JavaScript files.
+	xgettext --language=JavaScript --keyword=npgettext:1c,2,3 \
+		--from-code=utf-8 --sort-output --join-existing --no-wrap \
+		--package-name=$(shell node -e "console.log(require('./package.json').name);") \
 		--package-version=$(shell node -e "console.log(require('./package.json').version);") \
 		--output $@ $(GETTEXT_JS_SOURCES)
+# Generate .po files for each available language.
 	@for lang in $(LOCALES); do \
 		export PO_FILE=$(OUTPUT_DIR)/locale/$$lang/LC_MESSAGES/app.po; \
-		echo "msgmerge --sort-output --update $$PO_FILE $@"; \
+		echo "msgmerge --update $$PO_FILE $@"; \
 		mkdir -p $$(dirname $$PO_FILE); \
-		[ -f $$PO_FILE ] && msgmerge --lang=$$lang --sort-output --update $$PO_FILE $@ || msginit --no-translator --locale=$$lang --input=$@ -o $$PO_FILE; \
+		[ -f $$PO_FILE ] && msgmerge --lang=$$lang --update $$PO_FILE $@ || msginit --no-translator --locale=$$lang --input=$@ --output-file=$$PO_FILE; \
```

Comment:
I read this in the [`xgettext`](https://www.gnu.org/software/gettext/manual/html_node/xgettext-Invocation.html#xgettext-Invocation) doc about `--sort-output`:

> Note that using this option makes it much harder for the translator to understand each message’s context

Anyway the `--no-location` is used a little further:

> Note that using this option makes it harder for technically skilled translators to understand each message’s context

I don't know why you made this choice in the Systematic workflow?

As it is, I could put `--sort-output` back.